docs/mobile_data_collected.md

```diff
@@ -0,0 +1,173 @@
+|------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Session    | Session represents a real user journey on your mobile application. It begins when the user launches the application and it is kept live as long as the user stays active. During the user journey, all RUM events generated as part of the session will share the same `session.id` attribute.
+ |
+| View       | View represents a unique screen (or screen segment) on your mobile application. Individual ViewControllers are classified as distinct views. While a user stays on a view, RUM event attributes (Errors, Resources, Actions) get attached to the View with a unique `view.id`                                   |
+| Resource   | Resource represents network requests to first-party hosts, APIs, 3rd party providers and libraries in your mobile application. All requests generated during a user session are attached to the view with a unique `resource.id`                                                      |
+| Error     | Error represents an exception or crash (fatal error) emitted by the mobile application attached to the view it is generated in.                                                                                                                                            |
+| Action   | Action represents user activity in your mobile application (application launch, tap, swipe, back etc). Each action is attached with a unique `action_id` attached to the view it gets generated in.                                                                                                                                                |             
+
+The following diagram illustrates the RUM event hierarchy:
+
+{{< img src="real_user_monitoring/data_collected/event-hierarchy.png" alt="RUM Event hierarchy" style="width:50%;border:none" >}}
+
+## Default attributes
+
+RUM collects common attributes for all events and attributes specific to each event by default listed below. You can also choose to enrich your user session data with [additional events][1] to default events specific to your application monitoring and business analytics needs.
+
+
+### Common Core Attributes
+
+| Attribute name   | Type   | Description                 |
+|------------------|--------|-----------------------------|
+| `type`     | string | The type of the event (for example, `view` or `resource`).             |
+| `application.id` | string | The Datadog application ID. |
+| `session.id` | string | Unique ID of the session. |
+| `view.id` | string | Unique ID of the initial view corresponding to the event. |
+| `view.url` | string | URL of the initial view corresponding to the event. |
+| `view.name` | string | Name of the initial view corresponding to the event. |
```

Comment:
To make it more explanatory (`view.name` can be customized by the developer and it defaults to `view.url` if not):
```suggestion
| `view.name` | string | Custom name of the view corresponding to the event. |
```